hammerdb/run.tcl

```diff
@@ -19,7 +19,7 @@ diset tpcc pg_defaultdbase pguser
 diset tpcc pg_storedprocs true
 diset tpcc pg_driver timed
 diset tpcc pg_rampup 3
-diset tpcc pg_duration 200
+diset tpcc pg_duration 25
```

Comment:
I'd keep it 200 minutes, it could catch memory leaks etc. If anyone  ones a  quick  run, I think  he/she should lower this value. But, the  default that we'd  use in release testing seems better with 200

Reply:
In the hammerdb job on citus engine, we probably would want to have a quicker test, something like 20-30 mins run as in this PR. During release tests it makes more sense to have longer runs. We would hit some error with longer runs on hammerdb because of a bug in fab, which is probably fixed by https://github.com/citusdata/test-automation/pull/164.

We could programmatically change it in hammerdb job on citus engine to be around 20-30 mins and keep the default higher as well.

Reply:
@marcocitus's suggestion is to standardize on 60 mins I think.